django/thunderstore/api/cyberstorm/views/package.py

```diff
@@ -0,0 +1,155 @@
+from django.db.models import Count, QuerySet, Sum
+from django.http import Http404, HttpRequest, HttpResponse
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import status
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from thunderstore.api.cyberstorm.serializers import (
+    PackageCategoryCyberstormSerializer,
+    PackageDependencyCyberstormSerializer,
+    PackageDetailViewContentCyberstormSerializer,
+    PackageTeamCyberstormSerializer,
+    PackageVersionCyberstormSerializer,
+)
+from thunderstore.community.models import PackageListing
+from thunderstore.community.utils import get_preferred_community
+
+
+class PackageAPIView(APIView):
```

Comment:
Probably should be called `PackageDetailAPIView`

Reply:
No, I like PackageAPIView.

Reply:
That's a bit of a misleading naming convention since this view doesn't return a package resource, but rather a "virtual" resource for the package detail UI view that only exists as a concept. In other words, if & when we have a view that maps directly to the package db model, _that_ would be appropriately named as `PackageAPIView`, which would then create a duplicate class name (though not a conflict as it would likely be in a different module).

It's a non-blocker since it doesn't actually affect anything external, just noting on the inconsistency.